README.md

```diff
@@ -26,25 +25,28 @@ and then install the package
 
 ## Supported hypervisors
 
-virt-who can obtain list of guest running on given machine or association between hosts and guests in given virtualization environment.
+virt-who can obtain list of guest running on given machine or association between hosts and guests in given
+virtualization environment.
 
 Backends for reporting list of guest running on given machine:
+
 * libvirt
 
 Backends for reporting hosts to guests association:
+
 * RHV-M (4.x)
 * VMWare vCenter Server and ESX(i)
 * Hyper-V
 * Remote libvirt
 * Kubevirt
+* Nutanix AHV Virtualization
 
 
 ## Supported subscription managers
 
 virt-who can report the guest lists and host/guest associations to following subscription management systems:
 
-* Subscription Asset Manager (SAM)
-* Satellite 5
+* subscription.rhsm.redhat.com
```

Comment:
maybe "Red Hat Subscription Management (RHSM)" instead? i think this is generally used in other documentation

Reply:
Yeah, good idea.